express-zod-api/tests/integration.spec.ts

```
@@ -120,9 +105,12 @@ describe("Integration", () => {
   describe("Feature #1470: Custom brands", () => {
     test("should by handled accordingly", async () => {
       const rule: Producer = (
-        schema: z.ZodBranded<z.ZodTypeAny, PropertyKey>,
+        schema: ReturnType<z.ZodType["brand"]>,
         { next },
-      ) => next(schema.unwrap());
+      ) => {
+        delete schema._zod.bag.brand;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)